apps/landing-page/next.config.js

```diff
@@ -1,6 +1,15 @@
-module.exports = {
-  reactStrictMode: true,
-  images: {
-    domains: ["assets.coingecko.com", "pbs.twimg.com"],
```

Comment:
removing this will make the landing page break since images from exteral urls won't work anymore